### PR TITLE
Kaguya Driver Updates

### PR DESCRIPTION
**Pull request by Kelvinrr**
Things in this PR: 
* The driver now uses the correct key for line exposure duration. 
* 1/Pixel Pitch used in focal2pixel has it's signed inverted. 
* Now uses correct starting sample
* comments

With this, I think we can call the Kaguya driver in Ale complete until other problems arise? 

Some results using TC images with a ISIS3 Kaguya DEM as the base map: 
Screen Shot 2019-04-18 at 1 11 35 PM

### CODE REVIEW

File: ale/drivers/kaguya_driver.py [hunk 1]
```diff
@@ -129,21 +139,77 @@ def _odky(self):
...
+        LISM_TC2_SDN  (Single DCT Nominal)      297  3796(+4)  -131378
+        LISM_TC2_STN  (Single Through Nominal)  297  3504      -131379
+        LISM_TC2_WDH  (Double DCT Half)        1172  2921(+2)  -131380
+        LISM_TC2_WTH  (Double Through Half)    1172  2771      -131381
+        LISM_TC2_SDH  (Single DCT Half)        1172  2921(+2)  -131382
+        LISM_TC2_STH  (Single Through Half)    1172  2923      -131383
+        LISM_TC2_SSH  (Single SP_support Half) 1172  2921      -131384
+        """
+
+        return self.label["FIRST_PIXEL_NUMBER"]
```
Comment (kberryUSGS) [inline ale/drivers/kaguya_driver.py]: Just to make a note of another difference between ISIS and ALE, ISIS just uses 1 for FULL, 297 for NOMINAL, and 1172 for HALF. I would assume that `FIRST_PIXEL_NUMBER` is the same thing.

File: ale/drivers/kaguya_driver.py [hunk 2]
```diff
@@ -129,21 +139,77 @@ def _odky(self):
...
-            return self.label['LINE_EXPOSURE_DURATION'][0].value * 0.001  # Scale to seconds
-        else:
-            return self.label['LINE_EXPOSURE_DURATION'].value * 0.001  # Scale to seconds
+        """
+        Returns Line Exposure Duration
+
+        Kaguya TC
+        """
+        # It's a list, but only sometimes.
+        try:
```
[the remark this answers is not in the document]
Comment (Kelvinrr) [reply ale/drivers/kaguya_driver.py]: From TC1W2B0_01_02723N011E3240's label: 

```
CORRECTED_SAMPLING_INTERVAL          = (6.499967 <ms>)
```

Not sure if there is a rule for this. Line Exposure time was the same.

File: ale/drivers/kaguya_driver.py (hunk 1 above)
Comment (Kelvinrr) [reply ale/drivers/kaguya_driver.py]: yup, I originally had a lookup table for swath mode but it seems to have it baked into the label.

File: ale/drivers/kaguya_driver.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (kberryUSGS) [reply ale/drivers/kaguya_driver.py]: :+1: Thanks! I'll check it out to make sure we're handling both cases in ISIS.

**Comment by kberryUSGS** [comment]
@Kelvinrr Do you know why this is failing on Travis? Otherwise, it looks good to merge.

**Comment by Kelvinrr** [comment]
I forgot to put a colon in an else statement 😅

**Pull request merged**